mesonbuild/compilers/mixins/clike.py

```diff
@@ -266,6 +266,9 @@ def get_pch_use_args(self, pch_dir: str, header: str) -> T.List[str]:
     def get_pch_name(self, name: str) -> str:
         return os.path.basename(name) + '.' + self.get_pch_suffix()
 
+    def should_link_pch_object(self) -> bool:
+        return False
+
```

Comment:
This should be in the base `Compiler` class, we've been trying to not have specific languages have methods not supported in the base class.